Bottom: f9971c5b40daabda203772cd10e1c990126057fb
Top:    f0c7318a8172a9cbb87715ff0db503082547ab53
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-21 21:09:05 +0000

Turn off depth writing


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 4de2492..4b0a26f 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -992,6 +992,8 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
                 gport->bg_color.blue / 65535.,
                 1.);
 
+  glDepthMask (0);
+
   glClear (GL_COLOR_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
 
   region.X1 = MIN (Px (ev->area.x), Px (ev->area.x + ev->area.width + 1));
